src/constants/teams.ts

```diff
@@ -21,7 +21,41 @@ export const TeamCode: TeamCodeType = {
   nfo: "Nottingham Forest",
   shu: "Sheffield United",
   tot: "Tottenham Hotspur",
-  whu: "West Ham United",
+  whu: "West Ham",
   wol: "Wolverhampton Wanderers",
 };
+
+export type PremierLeagueTeam =
```

Comment:
You should be able to derive this array from the `values` of `TeamCode` above instead of re-writing it. 
Just want to keep the code so that if we need to change teams, codes, etc we only need to do it in one spot